python/requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 # Dependencies. When you update don't forget to update setup.py.
 pandas
-pyarrow>=4.0.0
```

Comment:
question: why are these removed?

Reply:
remove temporarily to see if this is causing the build to fail. I will add it again before the PR is completely ready.

please see my original comment regarding this.